core/src/main/resources/error/error-classes.json

```diff
@@ -80,6 +80,12 @@
     ],
     "sqlState" : "42000"
   },
+  "CANNOT_PARSE_VALUE_TO_DATATYPE" : {
+    "message" : [
+      "Cannot parse <value> to <valueType> type."
```

Comment:
`The value of the typed literal <valueType> is invalid: <value>.`

If `valueType` is `X`, the message might confuse since there is no type `X`.

Reply:
If `valueType` is `X`, should we translate it into `HEX` or others manually?